src/view/PrescriptionView.vue

```diff
@@ -1,9 +1,140 @@
+                        </div>
+                    </transition>
+                </teleport>
+
+                <button-component
+                    class="prescription__make_visit"
+                    @click="openModalImport = true">
+                    Importuj receptę
+                    <font-awesome-icon icon="fa-solid fa-plus" />
+                </button-component>
+
+                <teleport to="body">
+                    <transition name="modal">
+                        <div
+                            v-if="openModalImport"
+                            class="modal">
+                            <modal-component
+                                ref="modalImport"
+                                @close="openModalImport = false">
+                                <prescrition-import />
+                            </modal-component>
+                        </div>
+                    </transition>
+                </teleport>
+            </div>
+            <article class="prescription__listOf">
+                <prescription-group-component />
+            </article>
+
+            <article class="prescription__import">
+                <import-group-component />
+            </article>
+        </div>
+    </section>
 </template>
 
-<script setup></script>
+<script setup>
+    import ButtonComponent from '../components/ButtonComponent.vue';
+    import useUserStore from '../stores/userStore.js';
+    import ModalComponent from '../components/dashboard/ModalComponent.vue';
+    import PrescriptionGroupComponent from '../components/prescriptions/PrescriptionGroupComponent.vue';
+    import GreetingsComponent from '../components/GreetingsComponent.vue';
+    import PrescriptionAddComponent from '../components/prescriptions/PrescriptionAddComponent.vue';
+    import PrescritionImport from '../components/prescriptions/PrescritionImport.vue';
+    import ImportGroupComponent from '../components/prescriptions/ImportGroupComponent.vue';
+
+    import { onMounted, ref } from 'vue';
+    import { storeToRefs } from 'pinia';
+    import { onClickOutside } from '@vueuse/core';
+
+    const userStore = useUserStore();
+    const { getUserInfo } = storeToRefs(userStore);
+    const openModalPrescription = ref(false);
+    const openModalImport = ref(false);
+    const modalPrescription = ref(null);
+    const modalImport = ref(null);
+
+    onClickOutside(
+        modalPrescription,
+        () => (openModalPrescription.value = false)
+    );
+    onClickOutside(modalImport, () => (openModalImport.value = false));
+
+    onMounted(async () => {
+        await userStore.fetchUserData();
+        await userStore.fetchPatientPrescriptions();
+        await userStore.fetchPatientImportPrescriptions();
+    });
+</script>
 <style scoped lang="scss">
-    * {
+    .prescription {
+        position: relative;
+        width: 100%;
+        display: grid;
+        place-items: start;
+        grid-template-areas:
```

Comment:
good semantic grid area but change listOf to just list or list-of-refferals, cause as I remember css are broken when you write in grid-template-areas: area with camelCase notation